games.zetsubou.WanderNoMore.metainfo.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>games.zetsubou.WanderNoMore</id>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
+	<name>Wander No More</name>
+	<summary>A visual novel telling the story of a rogue samurai</summary>
+	<description>
+		<p>When Kouichirou Nabatame finally worked up the courage to desert his kingdom, he thought his days of bloodshed were behind him. He believed that if he ran far enough, and for long enough, he could leave it all behind. A normal life would finally be his, and he would be able to spend his days peacefully, helping others and protecting their happiness, rather than destroying them without mercy.</p>
+		<p>But as Kouichirou knew deep inside his heart, peace would not be his so easily.</p>
+		<p>After weeks on the run, Kouichirou passed through Hotsu Village, and the moment he did, he spelled its demise. The Kodokushi, an elite band of samurai he used to lead, slaughtered the village's residents, then set it ablaze and continued their search.</p>
+		<p>Powerless to stop them, Kouichirou emerged after his former subordinates moved on, desperate to put out the flames and find himself food and shelter. He moved from house to house, searching for anything edible, until he at last stumbled upon a storeroom, in which he found... a girl.</p>
+		<p>Orphaned, maimed, and robbed of her village, the girl would undoubtedly die a slow, painful death. Kouichirou knew this, and thus asked the girl a simple question.</p>
+		<p>"Do you want to live?"</p>
+		<p>But as he prepared to put the girl out of her misery, a fire lit in her eyes. Before she could vocalize her response, the girl, Chiyo, had already conveyed her answer.</p>
+		<p>"Then as the one responsible for all of this, it is my duty to make sure that happens. My name is Kouichirou Nabatame, and from this day onward, Chiyo, you are my daughter."</p>
```

Comment:
1. do you have permission to redistribute
2. You need  disclaimer that this in not official, etc.

Reply:
Metadata license has now been amended.

I do have permission to distribute. I haven't made the game open source, but I am the author of this game, and I intend for this to be an official distribution.

Reply:
so you are submitting the desltop file, icon and metainfo file upstream, aren't you? That's where they belong.

Reply:
Yes I am.